lib/node_modules/@stdlib/math/base/special/sincos/src/addon.c

```diff
@@ -0,0 +1,115 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
```

Comment:
I am a bit confused about how can we avoid writing the whole function here. We have done something like this in `rempio2`: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/rempio2/src/addon.c, but the difference is only that, over there, we have a `double` return value, here we aren't returning anything.

Reply:
You can do as we do in `rempio2`, with the exception that you don't need to create a `double` and can just return `NULL`.

Reply:
The implementation of `addon.c` in `rempio2` is almost equivalent to what you need to do here.